homeassistant/helpers/entity_registry.py

```diff
@@ -636,7 +636,6 @@ def _validate_item(
             unique_id,
             report_issue,
         )
-        return
```

Comment:
Should we convert it to a string?

Reply:
Absolutely not, the unique_id will then fail equality check